iati/core/tests/test_constants.py

```diff
@@ -13,3 +13,17 @@ def test_nsmap(self):
         """Check that the NSMAP constant is a dictionary with the correct key."""
         assert isinstance(iati.core.constants.NSMAP, dict)
         assert isinstance(iati.core.constants.NSMAP['xsd'], str)
+
+    def test_standard_versions_all_are_numbers(self):
+        """Check that each item in standard versions is a string that can be converted to a float."""
+        for version in iati.core.constants.STANDARD_VERSIONS:
+            assert isinstance(version, str)
+            assert float(version)
+
+    def test_standard_versions_correct_format(self):
```

Comment:
Another potential test for the STANDARD_VERSIONS - check that they're on the `Versions` Codelist (though this would break for unreleased versions... hum)

Reply:
Will add this as an issue - it's also dependent on work to download latest versions of codelists etc.

Reply:
https://github.com/IATI/iati.core/issues/125